frameworks/template/tests/conftest.py

```diff
@@ -1,8 +1,15 @@
 import pytest
 import sdk_security
 from tests import config
+from typing import List
 
+# Add the permissions you want to grant/revoke during the installation
+def additional_permissions(linux_user, service_account_name) -> List[dict]:
+    return []
 
 @pytest.fixture(scope='session')
 def configure_security(configure_universe):
-    yield from sdk_security.security_session(config.SERVICE_NAME)
+    linux_user = 'root'
+    service_account_name = 'service-acct'
```

Comment:
Maybe it would be more consistent if these values were moved to config.py as constants, e.g. `config.DEFAULT_LINUX_USER` or something like that? Also, does it make sense for `additional_permissions` to be a parameter of `configure_security` instead of a function that is called by it?

Reply:
Note that the `linux_user` used to be `nobody` and not `root`. Does this not change behaviour then?

Reply:
I'll change the linux user to `nobody`